plugins/modules/fusion_pp.py

```diff
@@ -128,14 +129,12 @@ def create_pp(module, fusion):
                 name=module.params["name"],
                 display_name=display_name,
                 objectives=[
-                    {
-                        "type": "RPO",
-                        "rpo": "PT" + str(module.params["local_rpo"]) + "M",
-                    },
-                    {
-                        "type": "Retention",
-                        "after": "PT" + str(local_retention) + "M",
-                    },
+                    purefusion.RPO(
+                        type="RPO", rpo="PT" + str(module.params["local_rpo"]) + "M"
```

Comment:
I believe here should be the parsed number as for the `local_retention`?
```suggestion
                        type="RPO", rpo="PT" + str(local_rpo) + "M"
```